develop/devguide/Automation/Howto/Getting_Started_with_the_IAS_Toolkit.md

```diff
@@ -88,6 +88,10 @@ public class Script
 }
 ```
 
+> [!NOTE]
+>
+> - As of DataMiner version 10.5.8 the comment ".FindInteractiveClient(" is no longer necessary, that can be defined on the [Interactivity](xref:DMSScript.Interactivity) tag of the script.
+
```

Comment:
As discussed, maybe it's worth adding a similar note to Building_interactive_Automation_scripts_with_CSharp to.

Reply:
@SkylineCommunications/team-docs-reviewers do you feel it's worth referring to the new option here?

Reply:
Yes, I think it might be worth mentioning there that the way you signal that a script is interactive has changed, with a reference to this page for more info.
Also, here on this page, I think it might be better to add two [tabs](https://docs.dataminer.services/contributing/CTB_Markdown_Syntax.html#tabs), one with the pre-10.5.8 version of the script example and one with the new example, to account for people who will simply copy it without looking at the note below it.